docs/root/start/quick-start/securing.rst

```diff
@@ -0,0 +1,199 @@
+.. _start_quick_start_securing:
+
+Securing Envoy
+==============
+
+Envoy provides a number of features to secure traffic in and out of your network, and
+between proxies and services within your network.
+
+Transport Layer Security (``TLS``) can be used to secure all types of ``HTTP`` traffic, including ``WebSockets``.
+
+Envoy also has support for transmitting and receiving generic ``TCP`` traffic with ``TLS``.
+
+.. warning::
+
+   The following guide takes you through individual aspects of securing traffic.
+
+   To secure traffic over a network that is untrusted, you are strongly advised to make use of encryption
+   and mutual authentication wherever you control both sides of the connection or where relevant protocols are available.
+
+   Here we provide a guide to using :ref:`mTLS <start_quick_start_securing_mtls>` which provides both encryption
+   and mutual authentication.
```

Comment:
We can do it as a follow up, but I think it would be good to cross link here to other authentication and authorization concepts such as JWT/RBAC/OAUTH/etc.

Reply:
cool, i have some other pending PRs around here, i reckon i can tack something on